src/printer/index.js

```diff
@@ -189,10 +150,29 @@ function printIf(path, print) {
     indent(concat([hardline, printBody(path, print)]))
   ];
 
-  const orElse = printOrElse(path);
+  while (n.orelse && n.orelse.length === 1 && n.orelse[0].ast_type === "If") {
+    n = n.orelse[0];
 
-  if (orElse) {
-    parts.push(orElse);
+    path = new FastPath(n);
+
+    parts.push(
+      hardline,
+      "elif ",
+      path.call(print, "test"),
+      ":",
+      indent(concat([hardline, printBody(path, print)]))
+    );
+  }
+
+  if (n.orelse && n.orelse.length > 0) {
+    parts.push(hardline, "else:");
+
+    for (let i = 0; i < n.orelse.length; i++) {
```

Comment:
can't you just do

```js
parts.push(indent(concat(path.map(print, 'orelse'))));
```

Reply:
for some reason it doesn't work, it is like it is missing the `indent` call:
```py
if foo:
  foo()
elif bar:
  bar()
else:baz()
```

Reply:
you need the `hardline`

Reply:
Why are we doing a manual `for` loop here rather than just re-using our `printBody` function, like we do the other times in this function?